android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -338,7 +338,9 @@ private void createViews() {
     @Override
     protected void onAttachedToWindow() {
         super.onAttachedToWindow();
-        initializePlayer();
+        if (player == null) {
```

Comment:
it might be a good idea to comment why we check for null in our fork. If it's a problem with react-native-video we should attempt to submit back issues to the parent repo.